airbyte-integrations/connectors/source-snapchat-marketing/source_snapchat_marketing/manifest.yaml

```diff
@@ -776,6 +776,8 @@ definitions:
               value: "{{ record.get('stats', {}).get('quartile_3') }}"
             - path: ["screen_time_millis"]
               value: "{{ record.get('stats', {}).get('screen_time_millis') }}"
+            - path: ["spend"]
```

Comment:
I believe so. Since this is just the `unnesting` operation, not sure why it takes place, instead of having the `stats` object described in schema. My best guess is this is leftovers after the migration to low-code since we attempted to match 1:1 with the old version.